src/main/java/or/sopt/houme/global/jwt/JWTFilter.java

```diff
@@ -58,10 +60,18 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
          * 토큰의 헤더가 access인지 확인해서 액세스토큰이 맞는지 검증한다
          * */
         try {
-            log.info("토큰 있어서 검증 시작");
 
             jwtUtil.isExpired(accessToken);
 
+            // 블랙리스트 검사
+            String jti = jwtUtil.getJti(accessToken);
+
+            if (blacklistTokenRepository.exists(jti)) {
+                setErrorResponse(response, ErrorCode.ACCESS_TOKEN_BLACKLISTED);
+                return;
+            }
```

Comment:
"// 블랙리스트 검사" 주석이 getJti에 있어서, 메서드 호출(getJti)과 조건문(exists) 둘 중 어느 부분을 설명하는지 모호해 보입니다. 또한, String jti를 추출하는 라인과 if 구문 코드 사이의 공백 한 줄은 제거해도 가독성에 도움이 될 것 같아요!

```suggestion
            // 블랙리스트 검사
            String jti = jwtUtil.getJti(accessToken);
            if (blacklistTokenRepository.exists(jti)) {
                setErrorResponse(response, ErrorCode.ACCESS_TOKEN_BLACKLISTED);
                return;
            }
```

Reply:
좋은 의견 감사드립니다! 코드의 가독성은 깊게 생각해보지 못한 부분이라서 놓쳤나봐요. 다음부턴 가독성도 신경써서 PR 준비하도록 하겠습니다!